spec/stat_tracker_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'spec_helper'
+
+RSpec.describe StatTracker do 
+  before(:each) do 
+    @game_path = './data/games.csv'
+    @team_path = './data/teams.csv'
+    @game_teams_path = './data/game_teams.csv'
+    
+    locations = {
+      games: @game_path,
+      teams: @team_path,
+      game_teams: @game_teams_path
+    }
+    
+    @stat_tracker = StatTracker.from_csv(locations)
+  end
+
+  describe "#Initialize" do 
+    it "exists w/ attribute" do 
+      expect(@stat_tracker).to be_a(StatTracker)
+      expect(@stat_tracker.games).to be_a(Array)
+      expect(@stat_tracker.teams).to be_a(Array)
+      expect(@stat_tracker.game_teams).to be_a(Array)
+    end
+  end
+
+  describe "#Create_games" do 
+    before(:each) do 
+      @game_data = {
+        away_goals: "2",
+        away_team_id: "3",
+        date_time: "5/16/13",
+        game_id: "2012030221",
+        home_goals: "3",
+        home_team_id: "6",
+        season: "20122013",
+        type: "Postseason",
+        venue: "Toyota Stadium",
+        venue_link: "/api/v1/venues/null"
+      }
+    end
+
+    it "can create games" do 
+      test_game = @stat_tracker.games.first
+
+      expect(test_game).to be_a(Game)
+      expect(test_game.away_goals).to eq(@game_data[:away_goals])
+      expect(test_game.away_team_id).to eq(@game_data[:away_team_id])
+      expect(test_game.date_time).to eq(@game_data[:date_time])
+      expect(test_game.game_id).to eq(@game_data[:game_id])
+      expect(test_game.home_goals).to eq(@game_data[:home_goals])
+      expect(test_game.home_team_id).to eq(@game_data[:home_team_id])
+      expect(test_game.season).to eq(@game_data[:season])
+      expect(test_game.type).to eq(@game_data[:type])
+      expect(test_game.venue).to eq(@game_data[:venue])
+      expect(test_game.venue_link).to eq(@game_data[:venue_link])
+    end
+  end
+  
+  describe "#Create_teams" do
+    it "can create a team" do 
+      team_data = {
+        abbreviation: "ATL",
+        franchise_id: "23",
+        link: "/api/v1/teams/1",
+        stadium: "Mercedes-Benz Stadium",
+        team_id: "1",
+        team_name: "Atlanta United"
+      }
+
+      test_team = @stat_tracker.teams.first
+
+      expect(test_team).to be_a(Team)
+      expect(test_team.abbreviation).to eq(team_data[:abbreviation])
+      expect(test_team.franchise_id).to eq(team_data[:franchise_id])
+      expect(test_team.link).to eq(team_data[:link])
+      expect(test_team.stadium).to eq(team_data[:stadium])
+      expect(test_team.team_id).to eq(team_data[:team_id])
+      expect(test_team.team_name).to eq(team_data[:team_name])
```

Comment:
rename `team_data` to `expected_team_data` to clarify test intentions

```suggestion
      expect(test_team.abbreviation).to eq(expected_team_data[:abbreviation])
      expect(test_team.franchise_id).to eq(expected_team_data[:franchise_id])
      expect(test_team.link).to eq(expected_team_data[:link])
      expect(test_team.stadium).to eq(expected_team_data[:stadium])
      expect(test_team.team_id).to eq(expected_team_data[:team_id])
      expect(test_team.team_name).to eq(expected_team_data[:team_name])
```